docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -0,0 +1,135 @@
+---
+id: publish-a-plugin
+title: Publish a plugin
+sidebar_position: 3
+description: "Learn how to package and share your plugin."
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - publish plugin
+  - update plugin
+---
+
+# Publish or update a plugin
+
+You've just built your plugin; now you want to share it with the world.
+
+In this guide, you'll learn how to package and share your plugin with others.
+
+The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog](https://grafana.com/plugins).
+By having your plugin published on Grafana.com, more users will be able to discover your plugin.
+
+## Before you begin
+
+When you build a plugin from source, a `dist` folder is created. This folder contains the production build or _plugin assets_ for your plugin.
+
+To package a plugin, refer to [Package a plugin](./package-a-plugin.md).
+
+### Follow our guidelines
+
+Get familiar with our plugin [publishing and signing criteria](./publishing-and-signing-criteria.md)
+
+### Do this for best results
+
+To speed up the time it takes to review your plugin:
+
+- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
+- Read our [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
```

Comment:
```suggestion
```

Reply:
Deleted. Follow up captured in https://github.com/grafana/plugin-tools/issues/360.